public/locales/en/common.json

```
@@ -243,34 +243,54 @@
     ]
   },
   "mentors": {
-    "title": "Mentors & Jurors"
+    "title": "Mentors & Jurors",
+    "roles": {
+      "jury": "Jury",
+      "mentor": "Mentor",
+      "speaker": "Speaker"
+    }
   },
   "schedule": {
     "title": "Schedule",
+    "info": "Details of the schedule are subject to change.",
     "days": [
       {
         "name": "04.11 - FRIDAY",
         "eventNames": {
-          "hackatonStart": "Hackaton starts",
-          "javaLecture": "Something about Java",
-          "graphQLforBegginers": "GraphQL Lecture",
-          "rememberToBeHydrated": "Time for drinking water 🍼"
+          "hackatonStart": "Start of the event",
+          "firstBreak": "Break",
+          "serviceLevelObjectives": "Lecture: Łukasz Dobek from Nobl9 - „Service Level Objectives”",
+          "secondBreak": "Break",
+          "modernAppPerformance": "Lecture: Piotr Jabłoński from WithSecure - „Performance of modern web applications - why bother with kilobytes in 2022?”",
+          "thirdBreak": "Break",
+          "accessibilityInApps": "Lecture: Jan Śmiełowski, Mikołaj Szymczak from All for one - „About accessibility, or how to write an application that is friendly to every user”",
```

Let's confirm the company names to not mistake them (even in terms of casing, some of them can be trademarked or be under some other regulations)
```suggestion
          "accessibilityInApps": "Lecture: Jan Śmiełowski, Mikołaj Szymczak from All For One - „About accessibility, or how to write an application that is friendly to every user”",
```